README.md

```diff
@@ -124,27 +128,149 @@ a proxy tool such as [mitmproxy](https://mitmproxy.org), or even just
 a script.
 
 A particularly simple way to proxy requests to a private bitcoind node
-is to make use of the [`corsproxy`](https://www.npmjs.com/package/corsproxy)
-npm module. Instructions to install and run the module are on its
-[home page](https://www.npmjs.com/package/corsproxy).
+is to make use of [`nginx`](https://nginx.org/). Instructions to install
+and run the program are on its [download page](https://nginx.org/en/download.html).
 
-Explicitly, install `corsproxy` with
+Explicitly, install `nginx` with
 
+```bash
+# MacOS
+brew install nginx
+
+# Debian Linux
+sudo apt install nginx
 ```
-$ npm install -g corsproxy
+
+Copy the server conifiguration file, `bitcoind.proxy`, to the appropriate location with the following
+commands. Note, these commands assume that you are in the base `caravan` directory. An example configuration
+file is included with the `caravan` source code called `bitcoind.proxy` which will, by defualt, enable a mainnet
+proxy. The testnet proxy is included, but is commented out.
+
+```bash
+# MacOS
+mkdir -p /usr/local/etc/nginx/sites-available
+cp bitcoind.proxy /usr/local/etc/nginx/sites-available/
+ln -s /usr/local/etc/nginx/sites-available/bitcoind.proxy /usr/local/etc/nginx/servers/bitcoind
+
+# Debain Linux
+sudo mkdir -p /etc/nginx/sites-available
+sudo cp bitcoin.proxy /etc/nginx/sites-available/
+sudo ln -s /etc/nginx/sites-available/bitcoin.proxy /etc/nginx/sites-enabled/bitcoind
```

Comment:
Note `bitcoind.proxy` here too.  Additionally I needed to create `/etc/nginx/sites-enabled`:

```
sudo mkdir /etc/nginx/sites-enabled
sudo ln -s /etc/nginx/sites-available/bitcoind.proxy /etc/nginx/sites-enabled/bitcoind
```

Reply:
Additionally the documentation I referred to for arch linux has config files in `/etc/nginx/sites-enabled` as `x.conf` instead of `x`.

Reply:
Made the changes and added some commentary regarding how to troubleshoot different distribution conventions.